approaches/byte-pair-encoding.md

```diff
@@ -5,3 +5,25 @@ layout: coming_soon
 title: Byte-pair encoding
 description: A data compression algorithm for machine translation
 ---
+
+Byte-pair encoding (BPE) is a data compression algorithm where the most common consecutive pair of bytes is replaced by a byte that is not seen in the data.
```

Comment:
Thanks for the article!

To be consistent with other articles, Byte-pair encoding should be formatted in bold.

> **Bite-pair encoding** (BPE) is...